src/docs/app-monitoring/sysdig-monitoring-guideline-for-shared-services-and-apps.md

```diff
@@ -0,0 +1,347 @@
+* [**Sysdig dashboard**](#sysdig-dashboard)
+* [**Uptime.com**](#uptimecom)
+* [**Runbook**](#runbook)
+* [**How to calculate error budget**](#how-to-calculate-error-budget)
+* [**Conclusion**](#conclusion)
+* [**Related pages**](#related-pages)
+
+---
+
+## BC Platform Services Product Registry
+The registry is an application that lets teams ask for namespaces in OpenShift 4 (OCP4) clusters. Here's what you can do with the registry:
+
+- Allowing teams to request the creation of new project namespaces in specific clusters
+- Enabling teams to update project contact information, manage resource quotas, and handle other metadata
+- Facilitating the request for access to various resources, including ACS, Vault, and Artifactory repositories
+- Allowing both the platform services team and AG to manage and supervise project sets.
+
+The technology stack for the registry comprises a React front-end, a Node.js backend, a MongoDB database, and an automation tool named "Provisioner."
+<br>
+<br>
+![Registry Flow Chart](../../images/registry-app-structure.png) 
+
+## Setting up SRE
+SRE involves deploying, configuring, and monitoring the app. It also includes ensuring services in production are available, managing latency, handling changes, responding to emergencies, and managing capacity of services in production. To ensure optimal performance and reliability, we employ various tools and methodologies that adhere to SRE principles.
+
+## Service Level Agreement (SLA)
+The customer should always be at the center of every aspect of your customer agreement. Even though an incident might involve addressing ten different issues on the back end, from the client's perspective, what truly matters is that the system operates as expected. Your SLAs and SLOs should reflect this reality. It's important to confine your commitments to high-level, user-facing functions and always use straightforward language in SLAs.
+
+Based on these principles, we can establish some SLAs for the Registry:
+
+* Normal users should be able to load the Registry dashboard successfully within 5 seconds.
+* Admin users should be able to load the Registry dashboard successfully within 13 seconds.
+* Approved product requests should be provisioned within an hour.
+* The application should be available online 99% of the time.
+* Updates to product requests should be processed within an hour.
+
+## Service Level Objective (SLO) 
+The meaning of an SLO is a service level objective: it's a specified target value or a range of values for a service level that gets measured through an SLI. Google offers a [valuable workbook](https://sre.google/workbook/implementing-slos/#:~:text=For%20example%2C%20if%20you%20have,50%25%20of%20the%20error%20budget.) written by Steven Thurgood, David Ferguson with Alex Hidalgo and Betsy Beyer to assist you in implementing SLOs. Towards the end of this document, we will also delve into calculating the Error Budget in more detail.
```

Comment:
```suggestion
SLO is a specified target value or a range of values for a service level that gets measured through an SLI. Google offers a [valuable workbook](https://sre.google/workbook/implementing-slos/#:~:text=For%20example%2C%20if%20you%20have,50%25%20of%20the%20error%20budget.) written by Steven Thurgood, David Ferguson with Alex Hidalgo and Betsy Beyer to assist you in implementing SLOs. Towards the end of this document, we will also delve into calculating the Error Budget in more detail.
```
in addition, can we avoid using acronyms before explaining what they are, i.e. `...gets measured through an SLI` while ppl might not know what SLI stand for. Or consider adding a link to them to point to a later section for reference?